From 1d7249f55e34a30cf8d6c34ad468e09483f76270 Mon Sep 17 00:00:00 2001
From: Tim Foster <tim.foster@joyent.com>
Date: Thu, 22 Nov 2018 17:43:22 +0000
Subject: [PATCH] MANTA-4021 wrasse's nodeunit dep cannot install with old node
 (fix build) Reviewed by: Trent Mick <trentm@gmail.com> Approved by: Trent
 Mick <trentm@gmail.com>

---
 Makefile     | 3 +--
 package.json | 3 ---
 2 files changed, 1 insertion(+), 5 deletions(-)

diff --git a/Makefile b/Makefile
index 74484bd..0992d47 100644
--- a/Makefile
+++ b/Makefile
@@ -30,7 +30,7 @@ NODEUNIT	:= ./node_modules/.bin/nodeunit
 #
 # Files
 #
-JS_FILES	:= $(shell ls *.js) $(shell find lib test -name '*.js')
+JS_FILES	:= $(shell ls *.js) $(shell find lib -name '*.js')
 JSL_CONF_NODE	 = tools/jsl.node.conf
 JSL_FILES_NODE   = $(JS_FILES)
 JSSTYLE_FILES	 = $(JS_FILES)
@@ -90,7 +90,6 @@ release: all docs $(SMF_MANIFESTS)
 		$(ROOT)/package.json \
 		$(ROOT)/sapi_manifests \
 		$(ROOT)/smf \
-		$(ROOT)/test \
 		$(RELSTAGEDIR)/root/opt/smartdc/$(NAME)
 	mv $(RELSTAGEDIR)/root/opt/smartdc/$(NAME)/build/scripts \
 	    $(RELSTAGEDIR)/root/opt/smartdc/$(NAME)/boot
diff --git a/package.json b/package.json
index da33c13..36027e4 100644
--- a/package.json
+++ b/package.json
@@ -23,8 +23,5 @@
         "rimraf": "2.2.6",
         "vasync": "1.4.3"
     },
-    "scripts": {
-        "test": "nodeunit test/*.test.js"
-    },
     "license": "MPL-2.0"
 }
-- 
2.21.0

